api/preprints/views.py

```diff
@@ -227,3 +232,145 @@ class PreprintContributorsList(NodeContributorsList, PreprintMixin):
     view_name = 'preprint-contributors'
 
     serializer_class = NodeContributorsSerializer
+
+
+class PreprintPreprintProvidersList(JSONAPIBaseView, generics.ListAPIView, ODMFilterMixin, NodeMixin):
+    """ Detail of the preprint providers a preprint has, if any. Returns [] if the preprint has no
+    preprnt providers.
+
+    ##Attributes
+
+    OSF Institutions have the "institutions" `type`.
+
+        name           type               description
+        =========================================================================
+        name           string             title of the institution
+        id             string             unique identifier in the OSF
+        logo_path      string             a path to the institution's static logo
+
+
+
+    #This Request/Response
+
+    """
+    permission_classes = (
+        drf_permissions.IsAuthenticatedOrReadOnly,
+        base_permissions.TokenHasScope,
+    )
+
+    required_read_scopes = [CoreScopes.NODE_PREPRINTS_READ]
+    required_write_scopes = [CoreScopes.NULL]
+    serializer_class = PreprintProviderSerializer
+
+    model = PreprintProvider
+    view_category = 'preprints'
+    view_name = 'preprint-preprint_providers'
+
+    def get_queryset(self):
+        node = self.get_node()
+        return node.preprint_providers or []
```

Comment:
`[]` is the default for a `ForeignField(list=True)`; this ternary is unnecessary